### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by jonri**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
> `--filesystem=host # we need read/write access to the game engine directory, which may be in any location`
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. [**Link: forum request**](https://forums.thedarkmod.com/index.php?/topic/21478-darkradiant-install-with-flatpak-linux/)
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
> *Domain is owned by another team member.*
- [x] Any additional patches or files have been submitted to the upstream projects concerned. 
> *Will commit appstream file in next released version.*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: net.darkradiant.DarkRadiant.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      - type: archive
+        url: https://github.com/codereader/DarkRadiant/archive/refs/tags/3.2.0.tar.gz
+        sha256: 3dc138cc0e203880e6eb514364af1b94589e34a8480cb7e2a2fc73d49e1523f3
+      - type: file
+        path: net.darkradiant.DarkRadiant.metainfo.xml
+    post-install:
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/darkradiant.desktop
+      - install -Dm644 ${FLATPAK_DEST}/share/darkradiant/bitmaps/darkradiant_icon_64x64.png ${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+cleanup:
```
Comment (hfiguiere) [inline net.darkradiant.DarkRadiant.yml:59]: all these are usually above modules.

File: net.darkradiant.DarkRadiant.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  - /lib/pkgconfig
+  - /lib/sigc++-2.0
+  - /lib/wx
+  - /share/aclocal
+  - /share/bakefile
+  - /share/eigen3
+  - /share/pkgconfig
+finish-args:
+  - --device=dri
+  - --filesystem=host # we need read/write access to the game engine directory, which may be in any location
```
Comment (hfiguiere) [inline net.darkradiant.DarkRadiant.yml:74]: that remain extremely broad filesystem permissions

File: net.darkradiant.DarkRadiant.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+# TODO: enable wayland/fallback-x11 once we are on wxgtk-3.2
+#  - --socket=wayland
+#  - --socket=fallback-x11
```
Comment (hfiguiere) [inline net.darkradiant.DarkRadiant.yml:80]:
```suggestion
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build net.darkradiant.DarkRadiant

File: net.darkradiant.DarkRadiant.yml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    config-opts:
+    sources:
+      - type: archive
+        url: https://github.com/codereader/DarkRadiant/archive/refs/tags/3.2.0.tar.gz
+        sha256: 3dc138cc0e203880e6eb514364af1b94589e34a8480cb7e2a2fc73d49e1523f3
+      - type: file
+        path: net.darkradiant.DarkRadiant.metainfo.xml
+    post-install:
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/darkradiant.desktop
+      - install -Dm644 ${FLATPAK_DEST}/share/darkradiant/bitmaps/darkradiant_icon_64x64.png ${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png
```
Comment (hfiguiere) [inline net.darkradiant.DarkRadiant.yml:57]: 128x128 at least is required.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
build failure
```
builddir/files/share/appdata/net.darkradiant.DarkRadiant.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• style-invalid         : Not enough <screenshot> tags, minimum is 1
Validation of files failed
```

**Comment by jonri** [comment]
bot, build net.darkradiant.DarkRadiant

**Pull request closed without merging**